irohad/multi_sig_transactions/impl/mst_processor_impl.cpp

```diff
@@ -88,11 +89,8 @@ namespace iroha {
   void FairMstProcessor::onNewState(const shared_model::crypto::PublicKey &from,
                                     MstState new_state) {
     log_->info("Applying new state");
```

Comment:
In next PR we need to fix everywhere 'MstState new_state' -> 'MstState&& new_state'. Because it should be moved in all 'onNewState' calls.